content/en/docs/contribute/new-content/open-a-pr.md

```diff
@@ -65,8 +65,7 @@ class id1 k8s
 
 Figure 1. Steps for opening a PR using GitHub.
 
-1. On the page where you see the issue, select the pencil icon at the top right.
-   You can also scroll to the bottom of the page and select **Edit this page**.
+1. On the page where you see the issue, select the **Edit this page** option in the right-hand side navigation panel.
 
```

Comment:
Aside: we could mention how to check for autogenerated pages.
See top right of https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/lease-v1/ for an example.

(best to address this in a separate PR)

Reply:
Hey @sftim ! Could you please explain about what exactly should I describe in opening a PR in auto-generated pages?

Reply:
I'm suggesting filing an issue, where we plan some work to make it more clear that when you are opening a pull request you first check to see if the page you are editing is automatically generated.

We should also provide advice on what to do if it is.